backend/app/schemas/quiz_check.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel
+from typing import List, Any
+
+
+class Smell(BaseModel):
+    """A class to represent a code smell in a file"""
+
+    start: int
+    end: int
+    type: str
+
+
+class File(BaseModel):
+    """A class to represent a file and its code smells"""
+
+    file_url: str
+    smells: List[Smell]
```

Comment:
The difference between built-in types and these from the `typing` module is subtle, but using the built-in is the preferred option.